ui/app/routes/application.js

```diff
@@ -140,7 +141,17 @@ export default class ApplicationRoute extends Route {
   @action
   error(error) {
     if (!(error instanceof AbortError)) {
-      this.controllerFor('application').set('error', error);
+      if (
+        error.errors?.any(
+          (e) =>
+            e.detail === 'ACL token expired' ||
+            e.detail === 'ACL token not found'
```

Comment:
`question`: Does this mean we're hard coding the error message that we're expecting from the API here? And thus, if they added a period to the end of the message, we wouldn't be able to match here?

Reply:
That's right — this is a pretty fragile way of doing a match. Luckily this PR also includes visual diff and acceptance tests for this situation, so if this ever changes we should be getting plenty of alarm bells.